src/js/components/Diagram/StyledDiagram.js

```diff
@@ -1,12 +1,44 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 import { defaultProps } from '../../default-props';
+import { animationObjectStyle } from '../../utils/animations';
+
+const animationItemStyle = (item, theme) => {
```

Comment:
Is `item` an animation type? if so, would it be more clear to call it `type` or `animationType`?